Sources/BraintreeCard/BTCardClient.swift

```diff
@@ -0,0 +1,218 @@
+
+                        if response?.statusCode == 422 {
+                            callbackError = self.constructCallbackError(with: error.userInfo, error: error)
+                        }
+
+                        self.sendAnalyticsEvent(with: false)
+                        completion(nil, callbackError)
+                        return
+                    }
+
+                    let cardJSON: BTJSON = body?["creditCards"][0] ?? BTJSON()
+                    self.sendAnalyticsEvent(with: !cardJSON.isError)
+
+                    // cardNonceWithJSON returns nil when cardJSON is nil, cardJSON.asError is nil when cardJSON is non-nil
+                    let cardNonce: BTCardNonce = BTCardNonce(json: cardJSON)
+                    completion(cardNonce, cardJSON.asError())
+                    return
+                }
+            }
+        }
+    }
+
+    /// Tokenizes a card
+    /// - Parameter card: The card to tokenize.
+    /// - Returns: On success, you will receive an instance of `BTCardNonce`
+    /// - Throws: An `Error` describing the failure
+    public func tokenize(_ card: BTCard) async throws -> BTCardNonce {
+        try await withCheckedThrowingContinuation { continuation in
+            tokenize(card) { nonce, error in
+                if let error {
+                    continuation.resume(throwing: error)
+                } else if let nonce {
+                    continuation.resume(returning: nonce)
+                }
+            }
+        }
+    }
+
+    // MARK: - Private Methods
+
+    private func isGraphQLEnabled(for configuration: BTConfiguration) -> Bool {
+        if let graphQLFeatures = configuration.json?["graphQL"]["features"].asStringArray() {
+            return !graphQLFeatures.isEmpty && graphQLFeatures.contains(graphQLTokenizeFeature)
+        }
+
+        return false
+    }
+
+    private func clientAPIParameters(for request: BTCardRequest) -> [String: Any] {
+        var parameters: [String: Any] = [:]
+        parameters["credit_card"] = request.card.parameters()
+
+        let metadata: [String: String] = [
+            "source": apiClient.metadata.sourceString,
+            "integration": apiClient.metadata.integrationString,
+            "sessionId": apiClient.metadata.sessionID
+        ]
+
+        parameters["_meta"] = metadata
+
+        if request.card.authenticationInsightRequested {
+            parameters["authenticationInsight"] = true
+            parameters["merchantAccountId"] = request.card.merchantAccountID
+        }
+
+        return parameters
+    }
+
+    // MARK: - Analytics
+
+    private func sendAnalyticsEvent(with success: Bool) {
+        let integration = apiClient.metadata.integrationString
+        let status = success ? "succeeded" : "failed"
+        let event = "ios.\(integration).card.\(status)"
+
+        apiClient.sendAnalyticsEvent(event)
+    }
+
+    private func sendGraphQLAnalyticsEvent(with success: Bool) {
```

Comment:
```suggestion
    private func sendGraphQLTokenizeAnalyticsEvent(with success: Bool) {
```
🧶

Reply:
These methods have both been removed in [`fpti-feature` branch](https://github.com/braintree/braintree_ios/compare/fpti-feature). Gonna just leave as is for now and resolve this entirely when we merge that branch in.